src/main/java/com/gistpetition/api/config/annotation/DataIntegrityHandler.java

```diff
@@ -0,0 +1,12 @@
+package com.gistpetition.api.config.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.METHOD)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface DataIntegrityHandler {
+    Class<? extends Exception> value();
```

Comment:
```suggestion
    Exception value();
```
이렇게는 안되나요? 안되는 이유 알거같긴 한데, 직접 듣고싶군여